src/lib/viewers/controls/media/MediaSettings/MediaSettingsAutoplay.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import MediaSettings, { Menu } from '.';
+
+export type Props = {
+    autoplay: boolean;
+    onAutoplayChange: (autoplay: boolean) => void;
+};
+
+export default function MediaSettingsMenuAutoplay({ autoplay, onAutoplayChange }: Props): JSX.Element {
+    const { setActiveMenu } = React.useContext(MediaSettings.Context);
+
+    const handleChange = (value: boolean): void => {
+        onAutoplayChange(value);
+        setActiveMenu(Menu.MAIN);
```

Comment:
Does it matter which order this is called?

Reply:
I don't think so, but I can invert the order since the `onAutoplayChange` callback will be handled by the viewer and `setActiveMenu` will be handled by React.